### PR TITLE
Last minute fixes

### PR DESCRIPTION
**Pull request by christina-cook**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Fixed style issues with the background image and posts. Fixed the styling of the loading page so that it's more clear to the user. Removed unnecessary comments and imports.

## *What are the relevant tickets?*
n/a

## *Concerns / Bugs:*
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Comment by christina-cook** [comment]
Screen Shot 2021-04-21 at 4 37 13 PM
This is nothing special but it's better than it was previously.

File: src/App/App.js [hunk 1]
```diff
@@ -53,10 +54,9 @@ const App = () => {
     { loading: loading2, error: error2, data: data2 }
   ] = QueryMultiple()
 
-  if (loading1 || loading2) return <p>Loading...</p>
-  if (data1) console.log('data 1', data1)
-  if (data2) console.log('data 2', data2)
-  if (error1 || error2) return <p>Error :(</p>
+  if (loading1 || loading2) return <div className='container'><div className='loading'><p>Loading</p><img src={loading} alt='loading' className='loading-img'/></div></div>
+  if (error1) return <div className='container'><p className='error'>Oops,something went wrong. Error: ${error1.message}</p></div>
+  if (error2) return <div className='container'><p className='error'>Oops,something went wrong. Error: ${error2.message}</p></div>
```
Comment (dietza) [inline src/App/App.js:59]: It's more code, but I feel like this is more effective for understanding what's happening here 👏

File: src/components/Feed/Feed.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const Feed = ({ profileIsVisible, searchResults }) => {
     return posts.sort((a, b) => {
       if (a.post.createdAt < b.post.createdAt) {
         return -1;
-      } else if (a.post.createdAt == b.post.createdAt) {
+      } else if (a.post.createdAt === b.post.createdAt) {
```
Comment (dietza) [inline src/components/Feed/Feed.js:28]: nice catch!

File: src/components/FormModal/FormModal.css [hunk 3]
```diff
@@ -17,7 +17,7 @@
 }
 
 .form__create-post input {
-  width: 110%;
+  width: 100%;
```
Comment (dietza) [inline src/components/FormModal/FormModal.css:20]: Nice, I made that same change on my branch 👍

**Review by dietza — APPROVED** [review]
Thanks for doing this clean up! and that loading screen was really bugging me - this new display is way better!

**Pull request merged**